include/vcpkg/base/messages.h

```diff
@@ -996,6 +1004,10 @@ namespace vcpkg
                     "msiexec failed while extracting '{path}' with launch or exit code {exit_code} and message:");
     DECLARE_MESSAGE(NoLocalizationForMessages, (), "", "No localized messages for the following: ");
     DECLARE_MESSAGE(NoRegistryForPort, (msg::package_name), "", "no registry configured for port {package_name}");
+    DECLARE_MESSAGE(PackageFailedtWhileExtracting,
+                    (msg::package_name, msg::path),
+                    "",
+                    "'{package_name}' failed while extracting '{path}'.");
```

Comment:
```suggestion
    DECLARE_MESSAGE(PackageFailedtWhileExtracting,
                    (msg::tool_name, msg::path),
                    "",
                    "{tool_name} failed while extracting {path}.");
```

1. More 's, 2. it looks like the users are passing the tool they did the extraction with here rather than talking about which package they're installing or building.